docs/how-to/download-real-time-sources.rst

```diff
@@ -0,0 +1,116 @@
+How to download Real-time Ubuntu kernel source code
+===================================================
+
+The source code for Ubuntu's real-time kernel is available for download.
+The following guide will show you how to download it for your system's release of Ubuntu.
+The process depends on the release of Ubuntu you are using.
+
+Enable access to the real-time Ubuntu kernel
+---------------------------------------------
+
+.. tab-set::
+    :sync-group: release
+
+    .. tab-item:: LTS releases (22.04, 24.04, etc.)
+        :sync: lts
+
+        .. note::
+            LTS releases of Ubuntu require Ubuntu Pro to download their respective real-time kernel source code.
+            See the `Ubuntu Pro documentation <https://documentation.ubuntu.com/pro/>`_ for more information.
+
+        If you've already enabled the real-time kernel, you don't need to do anything in this step.
+        You can verify the ``realtime-kernel`` service is activated with ``sudo pro status``.
+
+        Otherwise, enable access to the real-time kernel source packages with the following command:
+
+        .. code-block:: shell
+
+            sudo pro enable realtime-kernel --access-only
+
+
+    .. tab-item:: Interim releases (24.10, etc.)
+        :sync: interim
+        
+        The real-time kernel source is freely available in the 
+        `universe repository <https://packages.ubuntu.com/oracular/linux-realtime>`_ for interim releases. 
+        No actions are required for this step!
+    
+
+Enable downloading source packages with apt
+--------------------------------------------
+
+apt is used to download the real-time kernel source. We need to enable the source repositories (``deb-src``) to allow ``apt`` to download real-time kernel source packages from its archives.
+
+.. tab-set::
+    :sync-group: release
+
+    .. tab-item:: 22.04 LTS and older
+        :sync: lts
+
+        Open the ``/etc/apt/sources.list`` file in a text editor (requires elevated permissions) and :spellexception:`uncomment` (delete the leading ``#``) the line which includes ``deb-src http://archive.ubuntu.com/ubuntu/ <release> main``. 
+
+        For example, for Ubuntu 22.04 LTS (Jammy Jellyfish), only :spellexception:`uncomment` line number 4:
```

Comment:
22.04 was the first RT Ubuntu release. There is nothing older.
```suggestion
    .. tab-item:: 22.04 LTS
        :sync: lts

        Open the ``/etc/apt/sources.list`` file in a text editor (requires elevated permissions) and :spellexception:`uncomment` (delete the leading ``#``) the line which includes ``deb-src http://archive.ubuntu.com/ubuntu/ <release> main``. 
```
IMO repeating `Ubuntu 22.04 LTS (Jammy Jellyfish)` doesn't feel necessary here. Too many mentions of the same thing just confuses the reader.